actions/appSettings.ts

```diff
@@ -117,12 +106,32 @@ export async function storeEnvironment(formData: unknown) {
 
 export async function setSandboxMode(sandboxMode: boolean) {
   await setAppSetting('SANDBOX_MODE', sandboxMode);
-  safeRevalidateTag('getSandboxMode');
   return sandboxMode;
 }
 
 export async function setDisableAnalytics(disableAnalytics: boolean) {
   await setAppSetting('DISABLE_ANALYTICS', disableAnalytics);
-  safeRevalidateTag('getDisableAnalytics');
   return disableAnalytics;
 }
+
+export async function initializeWithDefaults() {
+  await prisma.$transaction(
+    async (tx) => {
+      // Use DEFAULT_APP_SETTINGS to initialize the app settings
+      for (const [key, value] of Object.entries(DEFAULT_APP_SETTINGS)) {
+        await tx.appSettings.create({
+          data: {
+            key: key as keyof typeof DEFAULT_APP_SETTINGS,
+            value:
+              typeof value === 'boolean'
+                ? value.toString()
+                : JSON.stringify(value),
+          },
+        });
+      }
+    },
+    {
+      timeout: 10000, // default is 5000 which is too short for this
+    },
+  );
+}
```

Comment:
Just thinking about this again....is there a reason not to use `createMany` in this case? Now we aren't calling separate functions, we don't need the transaction, right?

Reply:
resolved [5739d8d](https://github.com/complexdatacollective/Fresco/pull/161/commits/5739d8d788ef202113dca7f45afbaf157db5ba79)